scripts/generate-changelog.js

```diff
@@ -26,8 +33,8 @@ const changes = ({ version, from, to }) =>
 const run = async () => {
```

Comment:
Same here, I know it was already undocumented, but it's not straightforward, for example by defining the structure of `tags`, what do you think?